shub/settings/logging.py

```diff
@@ -0,0 +1,17 @@
+# Default Django logging is WARNINGS+ to console
+# so visible via docker-compose logs uwsgi
```

Comment:
I think this guy needs an `import os`

Reply:
and the logging object

Reply:
logging object is not needed - we aren't calling it. all the 'logging.' things are strings.

Reply:
yeah I didn't see in the interface that logging.StreamHandler is a string, hehe :) This is why I need my editor colors!